test/cr_multi_blocks_test.go

```diff
@@ -11,7 +11,7 @@ import "testing"
 // bob writes a multi-block file while unmerged, no conflicts
 func TestCrUnmergedWriteMultiblockFile(t *testing.T) {
 	test(t,
-		blockSize(20), users("alice", "bob"),
+		blockSize(20), blockChangeSize(20*1024), users("alice", "bob"),
```

Comment:
These two tests were causing us to use too many goroutines with `-race` using the default block change size, I think due to prefetching.